hacks/test-runner.sh

```diff
@@ -27,14 +27,14 @@ should_build() {
 }
 
 if [[ $TRAVIS_BRANCH == 'master' ]]; then
```

Comment:
This file can be deleted. We could adapt it for use with drone, but unit tests are running at an acceptable speed to not need this in the foreseeable releases.